h/static/scripts/annotator/guest.coffee

```diff
@@ -198,15 +201,37 @@ module.exports = class Guest extends Annotator
   _setupEditor: -> this
   _setupDocumentEvents: ->
     $(document).bind({
-      # omit the "mouseup" check
+      "touchstart": this.checkForStartSelection
       "mousedown": this.checkForStartSelection
     })
+
+    # This event is needed for iOS Safari only. 
+    if /iPhone|iPad|iPod/i.test(navigator.userAgent)
+      $(window).bind 'selectionEnd', ->
+        # reset selection timeout
+        selectionEndTimeout = null
+        # get user selection
+        selectedText = getSelectionText()
+        return
     this
 
+  # Needed to get text selection working on iOS. Only used for touch screen devices.
+  userSelectionChanged: ->
+    # Wait 500 ms after the last selection change event to capture 
+    if selectionEndTimeout
+      clearTimeout selectionEndTimeout
+    selectionEndTimeout = setTimeout((->
+      Util.getGlobal().trigger 'selectionEnd'
```

Comment:
It seems strange to trigger a custom event here only to catch it in exactly one other known place that's right above here in the code. Did you perhaps copy this from the answer http://stackoverflow.com/a/22787135/945546 has? I would just inline the code. Of course, that's assuming the code does anything (see comment on #L214)